style.css

```diff
@@ -0,0 +1,100 @@
+@import 'https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap';
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  background: #f1f1f1;
+}
+
+.main-header {
+  height: 50px;
+}
+
+.menu {
+  position: absolute;
+  left: 80%;
+  top: 4%;
+  font-size: 24px;
+}
+
+.main-header .menu {
+  left: 30;
+  line-height: 4;
+  display: flex;
+}
+
+.container {
+  background: #e5e5e5;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
-[ ] Please avoid setting any background color to the headline container and stick to the Figma design as much as possible 💯 

### Your Design:
![image](https://user-images.githubusercontent.com/68030297/226758125-df464205-da1f-474d-a219-24edffff80b3.png)

### Figma Design:
![image](https://user-images.githubusercontent.com/68030297/226758255-6f87caf3-2496-4017-9f06-ab29801a3e8a.png)